openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -466,12 +469,32 @@ class c04470(Variable):
     )
 
 
+class exemption_phaseout_start(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Exemption phaseout start"
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        return parameters(period).tax.income.exemption.phaseout.start[
+            tax_unit("mars", period)
+        ]
+
+
 class c04600(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
     documentation = """Personal exemptions after phase-out"""
 
+    def formula(tax_unit, period, parameters):
+        phaseout = parameters(period).tax.income.exemption.phaseout
+        phaseout_start = tax_unit("exemption_phaseout_start", period)
+        line_5 = max_(0, tax_unit("c00100", period) - phaseout_start)
+        line_6 = line_5 / (2500 / tax_unit("sep", period))
```

Comment:
Not exactly sure what the inherent meaning of `2500` is here, or where we should parametrise it

Reply:
https://github.com/PSLmodels/Tax-Calculator/issues/2631